paddlets/ensemble/ensemble_forecaster_base.py

```diff
@@ -186,7 +186,7 @@ def save(self, path: str, ensemble_file_name: str = "paddlets-ensemble-partial.p
         self._estimators = model_tmp
 
     @staticmethod
-    def load(path: str, ensemble_file_name: str = "paddlets-ensemble-partial.pkl") -> "EnsembleForecasterBase":
+    def load(path: str, ensemble_file_name: Optional[str] = "paddlets-ensemble-partial.pkl") -> "EnsembleForecasterBase":
```

Comment:
Line189 ensemble_file_name should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional